tests/builtins/test_divmod.py

```diff
@@ -8,27 +8,6 @@ class DivmodTests(TranspileTestCase):
 class BuiltinDivmodFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
     functions = ["divmod"]
 
-    not_implemented = [
```

Comment:
All these tests just work? Huh... That's... suspicious? Maybe that's true! If so, wow!